make_level.py

```diff
@@ -0,0 +1,104 @@
+import sys
+import os
+
+path = os.path
+join = os.path.join
+cwd = os.getcwd()
+
+def writeInit(skill_name, skill_path, level_name):
+    # If skills/skill/__init__.py doesn't exist, create a basic version (no levels)
+    if not path.exists(join(skill_path,"__init__.py")):
+        with open(join(skill_path,"__init__.py"), 'w+') as fp:
+            level_setup = "import pkg_resources\nfrom gitgud.skills.util import BasicLevel\nfrom gitgud.skills.util import Skill\n\nskill = Skill(\n\t'{}',\n\t[\n\t]\n)".format(skill_name)
```

Comment:
Separate this line out into multiple lines

```
level_setup = '\n'.join([
    "stuff",
    "morestuff",
    "", # blank line
])
```

Reply:
It might also be worth it to just read in a template file that is already all structured out, and then you just fill in the blanks with the brackets after reading it into a variable

Reply:
You can do `string = string.replace('\t', ' ' *4)`

Reply:
I replaced tabs with ' ' * 4 using the method described above, split up some of the string concatenations, but left some (the ones that only write one line to a file) let me know if that needs to be changed further.